.github/workflows/bigquery.yml

```diff
@@ -23,16 +26,31 @@ jobs:
         with:
           credentials_json: ${{ secrets.GOOGLE_CREDENTIALS }}
 
-      - uses: actions/setup-python@v4
+      # - uses: actions/setup-python@v4
+      #   with:
+      #     python-version: "3.x"
+      - uses: conda-incubator/setup-miniconda@v2
```

Comment:
Why this change? it looked like the action worked before this was committed?

Reply:
Nvm, I understand!

Reply:
Had to add this change because otherwise, the `conda` updating step of the `reindex.py` script didn't run, and removed all conda versions from all plugins. I'm not sure whether we should **also** keep the python setup step, because conda sets up its own python

Reply:
I removed the commented out python setup step in the latest commit